test/witan/models/dem/ccm/components_functions_test.clj

```diff
@@ -7,18 +7,58 @@
 
 (defn- fp-equals? [x y ε] (< (Math/abs (- x y)) ε))
 
-(def domestic-mig-in (ld/load-dataset :migration-mye
-                                      "resources/test_data/bristol_dom_in_mig.csv"))
+(def migration-data (ld/load-datasets {:domestic-in-migrants
+                                       "resources/test_data/migration/bristol_dom_in_mig.csv"
+                                       :domestic-out-migrants
+                                       "resources/test_data/migration/bristol_dom_out_mig.csv"
+                                       :international-in-migrants
+                                       "resources/test_data/migration/bristol_inter_in_mig.csv"
+                                       :international-out-migrants
+                                       "resources/test_data/migration/bristol_inter_out_mig.csv"}))
 
 (def dom-in-averages (ld/load-dataset :dom-in-averages
```

Comment:
add a comment to say these are averages generated in R to test results generated in Clojure